mgnify_pipelines_toolkit/analysis/assembly/add_rhea_chebi_annotation.py

```diff
@@ -0,0 +1,167 @@
+import argparse
```

Comment:
Please install the precommits and run them, the code needs to be format with `black`.

Pre-commit is part of the dev requirements.

```
pre-commit install
```
and then run
```
pre-commit run --all-files
```

This will reformat the code and run some other linters.